variables.tf

```diff
@@ -7,3 +7,9 @@ variable "target_url" {
   description = "URL to redirect to"
   type        = string
 }
+
+variable "origin_subdomain" {
```

Comment:
Maybe `source_domain`? Kinda matches the `target_url` variable.

The 'origin' part initially made me think it was referring somehow to the CloudFront Origin...